binderhub/templates/index.html

```diff
@@ -111,6 +97,40 @@ <h4 id="form-header">Build and launch a repository</h4>
             <div id="log"></div>
           </div>
         </div>
+
+        <!--url section-->
+        <div class="url">
+            <div class="bluedropdown">
+              <label>Copy and share this URL:</label>
+            </div>
+            <div class="url-row">
+              <pre id="basic-url-snippet"><a href="#" id="badge-url" title="show url snippet">Begin typing to see your URL!</a></pre>
+            </div>
+        </div>
+
+        <div class="badges">
+            <div class="bluedropdown" id="toggle-badge-snippet">
+              <label>Click, then paste into your README to get badge: <img id="badge" src="/static/images/badge.svg"></label>
```

Comment:
I think you might want to put `{{static_url(...)}}` here and a couple of places below.